templates/4a_compensate_image_data.ipynb

```diff
@@ -233,7 +233,10 @@
     "output_dir = os.path.join(rosetta_testing_dir, cohort_name, folder_name + '-stitched_with_' + current_channel_name)\n",
```

Comment:
<p>Are the different rosetta matrices generated with the multipliers saved or deleted? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>They're saved. Running <code>os.listdir(folder_path)</code> after this cell yields:</p><pre class="ql-syntax" spellcheck="false">['Noodle_all_commercial_rosetta_matrix_mult_1.csv',
 'Noodle_all_commercial_rosetta_matrix_mult_2.csv',
 'Noodle_all_commercial_rosetta_matrix_mult_0.5.csv'] 
</pre>